enc.go

```diff
@@ -24,7 +24,7 @@ func Encode(w io.Writer, stream *Stream) error {
 	enc := &encoder{bw: bitio.NewWriter(buf)}
 
 	// Store FLAC signature.
-	if _, err := enc.bw.Write(signature); err != nil {
+	if _, err := enc.bw.Write(flacSignature); err != nil {
```

Comment:
Haha, I forgot for a moment we have FLAC encoding. I was reading `enc` and thought, hmm. Shouldn't it be `dec`?

Well, we don't re-encode the samples yes, that is perhaps part of project *metalstorm* :)

Reply:
Haha, yeah I got the same reaction when I got the error. Like, "Wuah..? Ah yeah, now I remember."

However, the code we've written is very nice. I don't have that embarrased feeling I usually have when reading old code! :camel: :

Reply:
> However, the code we've written is very nice. I don't have that embarrased feeling I usually have when reading old code! :camel:

That's great to hear! I feel the same. Small things we can clean up here and there, e.g. #22, but mostly it is straightforward and consistent code.